README.md

```diff
@@ -70,6 +70,7 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 ## Contents
 
 - [Awesome Go](#awesome-go)
+  - [Contents](#contents)
```

Comment:
Everything looks good except for this line.

Reply:
@phanirithvij this is added by markdown lint, it's ok to add this line